src/strategy/gradual-rollout-random.ts

```diff
@@ -0,0 +1,14 @@
+import { Strategy } from '../strategy';
+import { Context } from '../context';
+
+export class GradualRolloutRandomStrategy extends Strategy {
+    constructor () {
+        super('gradualRolloutRandom');
+    }
+
+    isEnabled (parameters: any, context: Context) {
+        const percentage: number = +parameters.percentage;
```

Comment:
`Number(parameters.percentage)` is much cleaner than `+parameters.percentage` IMO